Malt/Shaders/Common/Transform.glsl

```diff
@@ -49,19 +49,19 @@ vec3 transform_normal(mat4 matrix, vec3 normal)
     return normalize(m * normal);
 }
 
-/* META @meta: category=Input; subcategory=Camera Info; */
+/* META @meta: category=Input; subcategory=Camera Info; internal=true;*/
```

Comment:
For files where everything or almost everything is internal, we should use a global meta property.

Reply:
got it. Will change.